pkg/api/user_token.go

```diff
@@ -53,9 +53,13 @@ func (hs *HTTPServer) getUserAuthTokensInternal(c *models.ReqContext, userID int
 
 	if err := hs.SQLStore.GetUserById(c.Req.Context(), &userQuery); err != nil {
 		if errors.Is(err, models.ErrUserNotFound) {
-			return response.Error(404, "User not found", err)
+			return response.Error(http.StatusNotFound, "User not found", err)
+		} else if errors.Is(err, models.ErrCaseInsensitive) {
+			return response.Error(http.StatusConflict,
```

Comment:
We block loading user tokens here in case of case insensitive logins. This causes a bit of inconsistency from my POV. It's still possible to load the user profile at `admin/users/edit/<userId>` page, but tokens are not loading with error. Is it expected behaviour?

Reply:
Yes, it will warn the admin there's a conflict but still allow username/email to be edited, in case:
 - a merge is not needed and one can be deleted
 - or they're users that require independent existence (different username/emails)